### DISCUSSION TITLE
Can mirrord be configured to control whether it's enabled or disabled within a monorepo setup?

### DISCUSSION THREAD

**Discussion by schwiftysmith** [metalbear-co/mirrord#2875] [Q&A] [answered]
I'm trying to set up a repository that includes both source code and end-to-end (e2e) tests.

When debugging, I'd like to run/debug the e2e tests without mirrord while using mirrord for debugging the source in the same IDE (e.g. VS Code). However, I haven’t found a straightforward way to disable mirrord by default for the e2e tests.

Is there a way to configure mirrord to be selectively enabled or disabled, such as through launch.json, environment variables, or another method in VS Code?

**Comment by aviramha** [comment, marked as answer]
It's not documented yet but you can use environment variable MIRRORD_ACTIVE=1 to explicitly enable mirrord regardless of button state. (Can't set it to 0 though)

**Reply by schwiftysmith** [reply]
Thanks for the quick answer @aviramha.
